seqr/views/require_anvil_reg_view.py

```diff
@@ -0,0 +1,6 @@
+from django.http import HttpResponse
+
+def require_anvil_registration(request, *args, **kwargs):
+    return HttpResponse('<p>Logging in has failed.</p><p>Make sure you have registered your account on AnVIL. Click '
+            '<a href=https://anvil.terra.bio>https://anvil.terra.bio</a> to sign in with Google and register your account.</p>'
+            '<a href=/login>Return</a>')
```

Comment:
Maybe, it should be redirected to a react page.

Reply:
I have changed the redirect URL. But I don't use the `match` property of the URL because the property is always empty.